.travis.yml

```diff
@@ -13,12 +13,14 @@ install:
   - eval $(opam config env)
   - opam install lablgtk camlp5 num --yes --verbose
 before_script:
-  - WPATH=$PWD
-  - cd ..        
-  - git clone https://github.com/UniMath/UniMath
-  - cd UniMath && git show && make PACKAGES="Foundations MoreFoundations Combinatorics Algebra NumberSystems CategoryTheory Topology Ktheory PAdics" install
+  - pushd ..        
+  - git clone --no-checkout https://github.com/UniMath/UniMath
+  - cd UniMath
+  - git checkout 2dadfb61f5ef0d9805cf0eb6b80ef2beb26472d5
```

Comment:
```suggestion
  - git checkout master
```

Reply:
I'd like to see how this works, building against master. If it does not work well, I am happy to revert to building against a fixed commit.